src/CFamily/Rules/RulesLoader.cs

```diff
@@ -40,7 +40,10 @@ public IEnumerable<string> ReadRulesList()
         {
             var rulesList = LoadCFamilyJsonFile<List<string>>("RulesList.json");
             Debug.Assert(rulesList != null, "The CFamily RulesList.json should exist and not be empty");
+            var misraRulesList = LoadCFamilyJsonFile<List<string>>("MisraRulesList.json");
+            Debug.Assert(rulesList != null, "The CFamily Misra RulesList.json should exist and not be empty");
 
+            rulesList.AddRange(misraRulesList);
```

Comment:
The CFamily pre-processor copies all of the MISRA files to the `lib` folder.
The only other thing we need to do is combine the lists of MISRA and non-MISRA rules so the rules provider knows the complete set of rules to look at.